MAUI/PDF-Viewer/Printing-PDF-files.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Printing PDF files in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here all about printing feature in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Printing PDF Files in .NET MAUI PDF Viewer (SfPdfViewer)
+
+![Printing PDF Files in .NET MAUI PDF Viewer](Images\Print\print.png)
+
+![Print Preview Window](Images\Print\print-preview-window.png)
+
+
+Using [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html), you can effortlessly print PDF documents directly from your application code. This feature enables users to conveniently fill PDF form fields and print the document with the updated values incorporated.
```

Comment:
* This does not help to fill the form fields. 
* Yesterday explained about this but could see some details are missing.

Reply:
Modified the details.